src/Kunstmaan/GeneratorBundle/Command/InstallCommand.php

```diff
@@ -58,16 +64,36 @@ protected function configure()
             );
     }
 
+    private function initAssistant($input, $output)
+    {
+        if (is_null($this->assistant)) {
+            $this->assistant = new CommandAssistant();
+            $this->assistant->setQuestionHelper($this->getQuestionHelper());
+            $this->assistant->setKernel($this->getApplication()->getKernel());
+        }
+        $this->assistant->setOutput($output);
+        $this->assistant->setInput($input);
+    }
+
     protected function interact(InputInterface $input, OutputInterface $output)
     {
+        $this->initAssistant($input, $output);
+
         $questionHelper = new QuestionHelper();
 
         $outputStyle = new SymfonyStyle($input, $output);
         $outputStyle->writeln('<info>Installing KunstmaanCms...</info>');
         $outputStyle->writeln($this->getKunstmaanLogo());
 
+        if (Kernel::VERSION_ID >= 40000 && null === $input->getOption('db-installed')) {
+            $this->shouldStop = !$this->assistant->askConfirmation('We need access to your database. Are the database credentials setup properly? (y/n)', 'y');
```

Comment:
@Devolicious  Maybe a nice to have here, is to inject the `DATABASE_URL` env var to show to the user. So they can easier validate if the correct db info is set.

Reply:
Not sure how I feel about showing User/Pass on screen with a command :)

Reply:
You're right, keep it like this!